guides/common/modules/proc_disabling-registration-to-insights.adoc

```diff
@@ -0,0 +1,21 @@
+[id='disabling-registration-with-insights_{context}']
+
+= Disabling Registration to Red Hat Insights
+
+After {Project} installation, you can choose to unregister or register Red Hat Insights as needed. For example, if you need to use {Project} in a disconnected environment, you can unregister `insights-client` from {ProjectServer}.
```

Comment:
I think the only thing missing here is what is the default. I assume by default it is registered since you mention unregistering it first?

Reply:
Good point. Fixed.